python/nano/tutorial/notebook/training/pytorch-lightning/pytorch_lightning_training_bf16.ipynb

```diff
@@ -0,0 +1,299 @@
+    "                                                               [0.229, 0.224, 0.225])])\n",
+    "    val_transform = transforms.Compose([transforms.Resize(256),\n",
+    "                                        transforms.CenterCrop(224),\n",
+    "                                        transforms.ToTensor(),\n",
+    "                                        transforms.Normalize([0.485, 0.456, 0.406],\n",
+    "                                                             [0.229, 0.224, 0.225])])\n",
+    "\n",
+    "    # apply data augmentation to the tarin_dataset\n",
+    "    train_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=train_transform, download=True)\n",
+    "    val_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=val_transform)\n",
+    "\n",
+    "    # obtain training indices that will be used for validation\n",
+    "    indices = torch.randperm(len(train_dataset))\n",
+    "    val_size = len(train_dataset) // 4\n",
+    "    train_dataset = torch.utils.data.Subset(train_dataset, indices[:-val_size])\n",
+    "    val_dataset = torch.utils.data.Subset(val_dataset, indices[-val_size:])\n",
+    "\n",
+    "    # prepare data loaders\n",
+    "    train_dataloader = DataLoader(train_dataset, batch_size=32)\n",
+    "    val_dataloader = DataLoader(val_dataset, batch_size=32)\n",
+    "\n",
+    "    return train_dataloader, val_dataloader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = MyLightningModule()\n",
+    "train_loader, val_loader = create_dataloaders()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _The definition of_ `MyLightningModule` _and_ `create_dataloaders` _can be found in the_ [runnable example](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/pytorch-lightning/pytorch_lightning_training_bf16.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use BFloat16 mixed precision for your PyTorch Lightning application, you could simply **import BigDL-Nano Trainer, and set** `precision` **to be** `'bf16'`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bigdl.nano.pytorch import Trainer\n",
+    "\n",
+    "trainer = Trainer(max_epochs=5, precision='bf16')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Note**\n",
+    ">\n",
+    "> BFloat16 mixed precision in PyTorch Lightning applications requires `torch>=1.10`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> ⚠️ **Warning**\n",
+    "> \n",
+    "> Using BFloat16 mixed precision with `torch<1.12` may result in extremely slow training."
```

Comment:
`torch>=1.10` is ok but `torch<1.12` is slow？so which version is suitable for BF16 training